### PR TITLE
Feature/iot 1364 direct url to tabs

### PR DESCRIPTION
**Pull request by fcv-iteratorIt**
Reworked gateway and application detail tabs to use routing

### CODE REVIEW

File: src/app/applications/datatarget/datatarget-tab/datatarget-tab.component.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="jumbotron--table">
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="jumbotron">
+        <div *ngIf="applicationService.canEdit else readOnly">
+          <app-top-bar-table [component]="true" [title]="'APPLICATION.DATATARGET-SHOW' | translate"
+                             [ctaLabelPrimary]="'FORM.CREATE-NEW-DATATARGET' | translate" [ctaRouterLinkPrimary]="'../datatarget-new'">
```
Comment (augusthjerrild) [inline src/app/applications/datatarget/datatarget-tab/datatarget-tab.component.html:7]: This looks a little weird, maybe a little auto-format? :-)

File: src/app/gateway/gateway-detail/gateway-detail.component.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
 table {
   width: 100%;
 }
+a:focus {
+
```
Comment (augusthjerrild) [inline src/app/gateway/gateway-detail/gateway-detail.component.scss:5]: Is this used?

**Review by augusthjerrild — APPROVED** [review]

File: src/app/admin/users/user-detail/user-detail.component.ts [hunk 3]
```diff
@@ -30,7 +30,7 @@ export class UserDetailComponent implements OnInit, OnDestroy {
   user: UserResponse;
   public backButton: BackButton = {
     label: '',
-    routerLink: '/admin/users',
+    routerLink: undefined,
```
[the remark this answers is not in the document]
Comment (fcv-iteratorIt) [reply src/app/admin/users/user-detail/user-detail.component.ts:33]: Changes the behavior of the back button from being a direct path to an actual "back" press

File: src/app/applications/applications-routing.module.ts [hunk 4]
```diff
@@ -1,51 +1,94 @@
...
+import { DatatargetTabComponent } from '@applications/datatarget/datatarget-tab/datatarget-tab.component';
+import { IotDeviceDetailsTabComponent } from '@applications/iot-devices/iot-device-detail/iot-device-details-tab/iot-device-details-tab.component';
+import { IotDeviceHistoryTabComponent } from '@applications/iot-devices/iot-device-detail/iot-device-history-tab/iot-device-history-tab.component';
+import { IotDeviceDataPacketsTabComponent } from '@applications/iot-devices/iot-device-detail/iot-device-data-packets-tab/iot-device-data-packets-tab.component';
+import { IotDeviceDownlinkTabComponent } from '@applications/iot-devices/iot-device-detail/iot-device-downlink-tab/iot-device-downlink-tab.component';
+
+const applicationRoutes: Routes = [
+  {
+    path: '',
+    component: ApplicationsComponent,
```
[the remark this answers is not in the document]
Comment (fcv-iteratorIt) [reply src/app/applications/applications-routing.module.ts:26]: It is

**Pull request merged**